CHANGELOG.md

```diff
@@ -12,6 +12,11 @@ FEATURES:
 * **New build command** `make test-env-build` can configure an empty vCD to run the test suite. See `TESTING.md` for details.
 * **New Resource:** Load Balancer Server Pool `vcd_lb_server_pool` - [GH-268]
 * **New Data Source:** Load Balancer Server Pool `vcd_lb_server_pool` - [GH-268]
+* **Resource:** Org VDC `vcd_org_vdc` update and full state read added - [GH-275]
+
+IMPROVEMENTS:
+* Fix ignoring of resource guarantee values in `vcd_org_vdc` [GH-265]
+* **Resource:** Org VDC `vcd_org_vdc` ID changed from name to vCD ID - [GH-275]
```

Comment:
`**Resource:**` highlight shouldn't be needed here. Unless I missed it, in guidelines there's no emphasis on improvements. They suggest using a form like this instead:

```
IMPROVEMENTS:

* resource/load_balancer: Add `ATTRIBUTE` argument (support X new functionality) [GH-12]
* resource/subnet: Now better [GH-22, GH-32]
```

Reply:
fixed